adapters/oidc/js/src/keycloak.js

```diff
@@ -445,6 +445,11 @@ function Keycloak (config) {
             url += '&ui_locales=' + encodeURIComponent(options.locale);
         }
 
+        if (options && options.acr) {
+            var claimsParameter = new ClaimsBuilder().acr(options.acr).build();
```

Comment:
I don't see why this needs to use a class, can we make this a function instead?

Reply:
@jonkoops I see your point.

My motivation was that OIDC specification has parameter called "claims", which is JSON structure, which can be sent from client application. See the details here: https://openid.net/specs/openid-connect-core-1_0.html#ClaimsParameter .

This PR contains just support for "acr" claim, but I was thinking that in the future, if we want to add better support for the "claims" parameter, we will just be able to re-use the ClaimsBuilder structure.

So that was motivation for introducing ClaimsBuilder, but at the same time, I don't have strong need to introduce it now in this PR. So if you prefer, I can remove `ClaimsBuilder` and use simple function for now, and then later, we can revisit and introduce some mechanism for more flexible adding of `claims` parameter.

WDYT? Are you ok to keep it like in the PR or do you prefer to simplify it and eventually re-add something for `claims` in some future PRs?

Reply:
I'd prefer to keep it simple at the moment, a function such as `buildClaimsParameter(options.acr)` would be fine for now.

Reply:
Ok, updated PR per this